sfyra/pkg/tests/server.go

```diff
@@ -177,6 +177,51 @@ func TestServerPatch(ctx context.Context, metalClient client.Client, talosInstal
 	}
 }
 
+// TestServerAcceptance makes sure the accepted bool works.
+func TestServerAcceptance(ctx context.Context, metalClient client.Client, vmSet *vm.Set) TestFunc {
+	return func(t *testing.T) {
+		numNodes := len(vmSet.Nodes())
+
+		servers := &v1alpha1.ServerList{}
```

Comment:
the only problem here is that this test is not idempotent if we re-run it. 

I don't have any good idea to keep it both idempotent and verify that accepted is set to false... except if we just create a mock server and verify accepted is false?